webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/KnoraGroupService.scala

```diff
@@ -10,17 +10,20 @@ import zio.Task
 import zio.ZIO
 import zio.ZLayer
 
+import dsp.errors.BadRequestException
```

Comment:
**issue**: `BadRequest` is an exception which should not be used in the domain package.

Reply:
Changes in [d1d97fc](https://github.com/dasch-swiss/dsp-api/pull/3195/commits/d1d97fc1a9aeaba6118f0c7be088207085d24ff5)